deltachat-ios/Helper/Constants.swift

```diff
@@ -20,15 +20,13 @@ struct Constants {
 }
 
 struct Time {
-    static let thirtySeconds = 30
     static let oneMinute = 60
-    static let twoMinutes = 2 * 60
     static let fiveMinutes = 5 * 60
-    static let thirtyMinutes = 30 * 6
+    static let thirtyMinutes = 30 * 60
```

Comment:
thirtyMinutes was wrongly defined before, however also not used (neither by ephemeral nor by mute)